tests/opf/test_solver.py

```diff
@@ -245,7 +231,9 @@ def test_dc(self):
             gen_original = solution_original["gen"][orig_ind]
             gen_reordered = solution_reordered["gen"][ind]
             for key, value in gen_original.items():
-                self.assert_approx_equal(gen_reordered[key], value, tol=1e-3)
+                # Some Gen entries are not initialized, e.g., Qc2min, check for nan
+                if value == value:
```

Comment:
Use [math.isnan](https://docs.python.org/3/library/math.html#math.isnan) to check for NaN.